modules/ocf_ceph/manifests/init.pp

```diff
@@ -0,0 +1,26 @@
+# Base class for ceph
+
+class ocf_ceph {
+  require ocf::packages::ceph
+
+  $ceph_fsid = lookup('ceph::fsid')
+
+  $ceph_mons_list = lookup('ceph::mons')
+  $ceph_mons = $ceph_mons_list.join(',')
+  $ceph_mon_ips = $ceph_mons_list.map
+    |$node| { ldap_attr($node, 'ipHostNumber') }.join(',')
+
+  $ip_addr = $::ipaddress
+
+  file { '/etc/ceph/ceph.conf':
+    content => template('ocf_ceph/ceph.conf.erb'),
+    require => Package['ceph'],
```

Comment:
I'd actually say to keep this as-is, since it makes it clear what this actually depends on (it needs the package to be installed first), and that dependency should survive even if this resource is moved to another file, etc.

I'd agree it could technically be removed, but I think keeping it is better to be more explicit about dependencies and not rely on the implicit behavior of puppet to run in manifest order. Thoughts?